splash-lp-indexer/src/position_db/pool_frames.rs

```diff
@@ -0,0 +1,35 @@
+use async_trait::async_trait;
+use crate::position_db::{PositionDB, ACCOUNTS_CF, POOL_LQ_FRAMES_INDEX_CF};
+use rocksdb::ReadOptions;
+use spectrum_offchain_cardano::data::PoolId;
+use tokio::task::spawn_blocking;
+
+#[async_trait]
+pub trait PoolFrames {
+    async fn get_pool_lq_supply(&self, pool_id: PoolId) -> Option<u64>;
+}
+
+#[async_trait]
+impl PoolFrames for PositionDB {
+    async fn get_pool_lq_supply(&self, pool_id: PoolId) -> Option<u64> {
+        let db = self.db.clone();
+        spawn_blocking(move || {
+            let tx = db.transaction();
```

Comment:
Why you need a TX to read a single value?

Reply:
no necessary, will fix

Reply:
fixed